eos-updater-prepare-volume/eos-updater-prepare-volume

```diff
@@ -143,6 +148,40 @@ class VolumePreparer:
 
         return autoinstalls
 
+    def _ostree_get_lock_timeout(self):
+        """
+        Retrieve the OSTree lock timeout.
+
+        This determines how long commands like 'ostree summary --update' and
+        'ostree create-usb' will wait to get an exclusive lock on the repo
+        before failing.
+        """
+
+        timeout_s = self.OSTREE_LOCK_TIMEOUT_S_DEFAULT
+        try:
+            out = subprocess.check_output(['ostree', 'config', 'get',
+                                          'core.lock-timeout-secs'])
+            timeout_s = float(out)
```

Comment:
The timeout is an integer, not a float.

Reply:
It actually seems to be an arbitrary value based on `ostree-repo.c` but I'd be fine with treating it as an `int` here.

Reply:
Fixed and tested in the version I just pushed